content/hi/docs/contribute/review/reviewing-prs.md

```diff
@@ -0,0 +1,150 @@
+---
+title: पुल अनुरोधों की समीक्षा करना
```

Comment:
```suggestion
title: पुल रिक्वेस्ट रिव्यु करना
```

Reply:
I would rather rephrase this as पुल रिक्वेस्ट की समीक्षा करना.

The reason is that pull request/PR is nomenclature very specific to GitHub's version control mechanism. Reviewing, on the other hand, is a verb and can be translated. Thoughts?